package.json

```diff
@@ -100,6 +103,7 @@
   },
   "homepage": "https://Addepar.github.io/ember-table",
   "resolutions": {
+    "@ember/test-waiters": "^3.0.2",
```

Comment:
ember/test-waiters at 3.0.1 is not compatible with embroider. We fix this using a resolution because it's a secondary dependency of ember-a11y-testing, and bumping ember-a11y-testing to the ^5.2.1 range did not pick up the newer ember/test-waiters.
In a follow-up I'll attempt to upgrade ember-a11y-testing, at which point we can probably remove this.